libc/src/__support/FPUtil/BasicOperations.h

```diff
@@ -64,16 +64,14 @@ LIBC_INLINE T fmaximum(T x, T y) {
 
   if (bitx.is_nan()) {
     return x;
-  } else if (bity.is_nan()) {
+  }
+  if (bity.is_nan()) {
```

Comment:
Remove the curly braces `{}` around a single statement after `if`.  Same as above and below.

Reply:
ok, I will do that. thanks